python/ray/rllib/bc/policy.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
```

Comment:
In the spirit of reducing repetition, this file could probably be extended from `a3c/shared_model.py` with a modification of the `setup_loss`

Reply:
indeed, they should both be external to any specific algo; this is one of the issues I wanted to open

Reply:
yeah, the model/framework-dependent code hasn't been refactored yet.